src/CCFieldValidator.js

```diff
@@ -2,13 +2,23 @@ import valid from "card-validator";
 import pick from "lodash.pick";
 import values from "lodash.values";
 import every from "lodash.every";
+import { CARDS_OVERRIDES } from "./cardsOverrides";
+
+const MAESTR0_BASE_LENGTH = 12;
+CARDS_OVERRIDES.forEach(card => valid.creditCardType.addCard(card));
 
 const toStatus = validation => {
   return validation.isValid ? "valid" :
          validation.isPotentiallyValid ? "incomplete" :
          "invalid";
 };
 
+const hasProperLength = cardNumber => cardNumber.split(" ").join("").length >= MAESTR0_BASE_LENGTH;
+const maestroCardStatus = (numberValidation, cardNumber) =>
+  hasProperLength(cardNumber)
+  && numberValidation.card
+  && numberValidation.card.type === "maestro" && "valid";
```

Comment:
One question:  why was adding the card to card-validator not enough ?

I think you can pass the validator an array of valid lengths in addCard, 
and I don't think maestro has cards with more than a thousand digits.

I would take back these changes and use an array of valid lengths as long as needed. (No need to write it by hand, you can just loop it).